db/column_family_test.cc

```diff
@@ -3091,9 +3100,9 @@ TEST_P(ColumnFamilyTest, CompactionSpeedupForMarkedFiles) {
   ASSERT_EQ(kParallelismLimit, dbfull()->TEST_BGCompactionsAllowed());
```

Comment:
When I run UT(CompactionSpeedupForMarkedFiles) locally, this line often fails to pass, and the reason is the current test case will share the same env with another case that may have already increased the number of background threads. So I make a simple fix